script.js

```diff
@@ -0,0 +1,87 @@
+const submitForm = document.querySelector('.input-box'); // 할 것 제출
+const button = document.querySelector('button'); // 입력버튼
+const textInput = document.querySelector('input'); // 추가 할 내용
+const todolist = document.querySelector('#todo-list'); // 버튼을 눌렀을 때 추가되는 곳
+const todo = document.querySelector('.todo');
+const done = document.querySelector('.done');
+const doneList = document.querySelector('#done-list');
```

Comment:
좋은 것 같아요.
todo, done은 아이디가 있으니까(`#done-header`) 아이디로 선택해주시면 좋을 것 같아요. 순서도 정리해주시고, 변수 이름도 좀 더 구체적으로 지어주세요!
```suggestion
const submitForm = document.querySelector('.input-box'); // 할 것 제출
const button = document.querySelector('button'); // 입력버튼
const textInput = document.querySelector('input'); // 추가 할 내용
const todolist = document.querySelector('#todo-list'); // 버튼을 눌렀을 때 추가되는 곳
const todoListHeader = document.querySelector('#todo-header');
const doneList = document.querySelector('#done-list');
const doneListHeader = document.querySelector('#done-header');
```

Reply:
순서 정리도 하고 코드도 다시 볼 수 있도록 깔끔하게 하겠습니다!!